src/frontend/mame/ui/menu.cpp

```diff
@@ -341,8 +357,8 @@ void menu::item_append(const char *text, const char *subtext, UINT32 flags, void
 
 	// allocate a new item and populate it
 	menu_item pitem;
-	pitem.text = (text != nullptr) ? pool_strdup(text) : nullptr;
```

Comment:
ah sorry was thinking of object_pool

On Wed, Jun 29, 2016 at 2:46 PM, Miodrag Milanovic <mmicko@gmail.com> wrote:

> pool is still used, one of reason of redoing hashfile.cpp has removing
> pool usage
>
> On Wed, Jun 29, 2016 at 2:39 PM, Vas Crabb <notifications@github.com>
> wrote:
>
>> In src/frontend/mame/ui/menu.cpp
>> <https://github.com/mamedev/mame/pull/1011#discussion_r68937985>:
>>
>> > @@ -341,8 +357,8 @@ void menu::item_append(const char *text, const char *subtext, UINT32 flags, void
>> >
>> >  	// allocate a new item and populate it
>> >  	menu_item pitem;
>> > -	pitem.text = (text != nullptr) ? pool_strdup(text) : nullptr;
>>
>> Is the pool used for anything else now that this is gone?
>>
>> —
>> You are receiving this because you are subscribed to this thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/mamedev/mame/pull/1011/files/78576d09c66b7bfb6d3e6d41c38fc9c059e6cb6b#r68937985>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/ADdKSMUr2UONhduDElXmJ_FLxa4NDkMhks5qQmeDgaJpZM4JBFBb>
>> .
>>
>
>